voting-contentApp/apps/voting-app.yaml

```diff
@@ -0,0 +1,31 @@
+!content-type:poll
+name: voting-app
+label: Voting
+icon: icon-segments-app
+
+subApps:
+  browser:
+    workbench:
+      contentViews: !override
+        - name: tree !override
+          multiSelect: true
+          $type: treeView
+          dropConstraint:
+            $type: jcrDropConstraint
```

Comment:
do we need this?

Reply:
Do you mention the dropConstraint property? It's optional to drop and down the row. Should I delete it cause it was not mentioned in the requirement?

Reply:
If it's working fine, leave it there.